pallets/subtensor/src/coinbase/root.rs

```diff
@@ -1150,13 +1249,13 @@ impl<T: Config> Pallet<T> {
         // --- 4. Remove netuid from added networks.
         NetworksAdded::<T>::remove(netuid);
 
-        // --- 6. Decrement the network counter.
-        TotalNetworks::<T>::mutate(|n| *n = n.saturating_sub(1));
+        // --- 5. Decrement the network counter.
```

Comment:
I suggest removing these numbers, they can be annoying to maintain

Reply:
const is very opinionated about having these